Networking/NetworkingTests/Requests/AuthenticatedDotcomRequestTests.swift

```diff
@@ -0,0 +1,94 @@
+import Foundation
+import XCTest
+import WordPressShared
+@testable import Networking
+
+/// AuthenticatedDotcomRequest Unit Tests
+///
+final class AuthenticatedDotcomRequestTests: XCTestCase {
+
+    /// Sample Unauthenticated Request
+    ///
+    private var unauthenticatedRequest: URLRequest!
+
+    /// Sample Credentials
+    ///
+    private let credentials = Credentials(username: "username", authToken: "yosemite", siteAddress: "https://wordpress.com")
+
+    override func setUp() {
+        super.setUp()
+
+        unauthenticatedRequest = try! URLRequest(url: "www.automattic.com", method: .get)
+    }
+
+    override func tearDown() {
+        unauthenticatedRequest = nil
+
+        super.tearDown()
+    }
+
+    /// Verifies that the Bearer Token is injected, as part of the HTTP Headers.
+    ///
+    func test_bearer_token_is_injected_as_request_header_when_authenticated_using_WPCOM_token() {
+        // Given
+        XCTAssertEqual(unauthenticatedRequest.allHTTPHeaderFields, [:])
+
+        guard case let .wpcom(_, authToken, _) = credentials else {
+            XCTFail("Missing credentials.")
+            return
+        }
+
```

Comment:
nit: do we still need `Credentials`? it looks like we can just pass an auth token string to the next line directly.

Same for other test cases in this file.

Reply:
Done in c66416dffcf5c28760675b5d4abae33a1b4d24d5